html/pfappserver/lib/pfappserver/Form/Config/SelfService.pm

```diff
@@ -49,10 +49,11 @@ has_field 'roles_allowed_to_unregister' =>
 has_field 'device_registration_role' =>
   (
    type => 'Select',
+   multiple => 1,
    label => 'Role to assign',
    options_method => \&options_roles,
    tags => { after_element => \&help,
-             help => 'The role to assign to devices registered from the self-service portal. If none is specified, the role of the registrant is used.' },
+             help => 'The role to assign to devices registered from the self-service portal. If none is specified, the role of the registrant is used. If multiples are defined then the user will have to choose' },
```

Comment:
The help needs to be adjusted in the frontend code, this here isn't honored anymore